src/Cache.ts

```diff
@@ -14,11 +14,24 @@ export class LocalStorageCache implements IConfigCatCache {
     try {
       const configString = localStorage.getItem(key);
       if (configString) {
-        return atob(configString);
+        return this.b64DecodeUnicode(configString);
       }
     }
     catch (ex) {
       // local storage is unavailable or invalid cache value in localstorage
     }
+    return void 0;
+  }
+
+  private b64EncodeUnicode(str: string): string {
```

Comment:
Does it make sense to move these methods into `js-common` ?

Reply:
It is not used in Node :D